rust/sealevel/libraries/account-utils/Cargo.toml

```diff
@@ -7,18 +9,17 @@ edition = "2021"
 no-entrypoint = []
 
 [dependencies]
-borsh = "0.9"
-solana-program = "=1.14.13"
+borsh.workspace = true
+solana-program.workspace = true
+spl-type-length-value.workspace = true
 
-[dependencies.spl-type-length-value]
-version = "0.1.0"
-git = "https://github.com/tkporter/eclipse-program-library"
-rev = "2f7e864"
-features = ["borsh"]
+hyperlane-core = { path = "../../../hyperlane-core" }
```

Comment:
nit: I think I added this to test things breaking - should be able to remove `hyperlane-core` as a dependency